src/ccip/views.py

```diff
@@ -169,8 +169,8 @@ def _transform_session(request, event, type_key, info_getter):
     session = {
         'id': f'{type_key}-{event_info.pk}',
         'type': type_key,
-        'start': event.begin_time.value.isoformat() if event.begin_time else None,
-        'end': event.end_time.value.isoformat() if event.end_time else None,
+        'start': event.begin_time.value.astimezone().isoformat() if event.begin_time else None,
+        'end': event.end_time.value.astimezone().isoformat() if event.end_time else None,
```

Comment:
Better to specify the +8 timezone explicitly.

Reply:
Fixed. Please take another look, thanks!